sawtooth-sdk-transaction-processor/src/main/java/sawtooth/sdk/processor/Context.java

```diff
@@ -28,7 +28,6 @@
 
   /**
    * Make a Get request on a specific context specified by contextId.
-   *
    * @param addresses a collection of address Strings
    * @return Map where the keys are addresses, values Bytestring
```

Comment:
I know this is a preexisting error, but it bothers me the comments/docstrings use `Bytestring` in several places. It should be `ByteString` and capitalizing it correctly should avoid some stupid mistakes/extra build cycles.

Reply:
believe I've corrected where that is a problem now.